ui/src/cloud/utils/reporting.ts

```diff
@@ -31,21 +31,14 @@ export const updateReportingContext = (properties: KeyValue) => {
   reportingTags = {...reportingTags, ...properties}
 }
 
-export const reportEvent = ({
-  timestamp = toNano(Date.now()),
-  measurement,
-  fields,
-  tags,
-}: Point) => {
+const pooledEvent = ({timestamp, measurement, fields, tags}: Point) => {
   if (isEmpty(fields)) {
     fields = {source: 'ui'}
   }
 
-  fireEvent(measurement, {...reportingTags, ...tags})
-
   reportingPoints.push({
-    measurement,
-    tags: {...reportingTags, ...tags},
+    measurement: 'UI Event', // artifact of sharing the tools cluster
```

Comment:
this means that every time we add a new call to `event`, we add a new tag to the measurement `UI Event`. This gives us no option to change the measurement and we just have to add tags any time we want to measure something? 

Why does everything need to be an event keyed off a single measurement? Were we asked not to create too many measurements?  What if we want to write metrics of the form:

```
{
  measurement: 'the thing i am measuring',
  tags: {...relevantTags},
  fields: {count: 1}
}
```
what is wrong with this kind of measuring?

Reply:
Nothing is wrong. But since all metrics are going to same `apps` bucket in tools, I asked that we have a uniform measurement to visually remove the things we are not interested in. @drdelambre I wonder if a better way for us to do this, could be to use the  `source: ui` tag we are placing on all points?

Reply:
@ebb-tide could we solve both concerns by changing the bucket that the data is written to?

Reply:
That would be great!